pkg/collector/corechecks/snmp/internal/report/report_utils.go

```diff
@@ -197,3 +197,10 @@ func getConstantMetricValues(mtcl checkconfig.MetricTagConfigList, values *value
 	}
 	return constantValues
 }
+
+// isInterfaceTableMetric takes in an OID and returns
+// true if the prefix matches ifTable or ifXTable from
+// the IF-MIB
+func isInterfaceTableMetric(oid string) bool {
+	return strings.HasPrefix(oid, "1.3.6.1.2.1.2.2") || strings.HasPrefix(oid, "1.3.6.1.2.1.31.1.1")
```

Comment:
The OID here is from the profile table metrics.

I think if we want to handle it, it might better to do it earlier in validation/normalisation steps here: https://github.com/DataDog/datadog-agent/blob/ce45fa68d7fb740c568b1a700a3d0558d5b83a92/pkg/collector/corechecks/snmp/internal/checkconfig/config_metric.go#L161-L176

Reply:
Having a redundant check doesn't hurt anything, but yes I agree this could be useful. To avoid coupling these changes (in the event something doesn't work with it) I'll open a separate PR for this change.

https://github.com/DataDog/datadog-agent/pull/17674